Examples/Python/src/Json.cpp

```diff
@@ -168,6 +168,16 @@ void addJson(Context& ctx) {
 
     mex.def("readSurfaceVectorFromJson",
             ActsExamples::JsonSurfacesReader::readVector);
+
+    py::class_<Acts::JsonDetectorElement, Acts::DetectorElementBase,
+               std::shared_ptr<Acts::JsonDetectorElement>>(
+        mex, "JsonDetectorElement")
+        .def("surface", [](Acts::JsonDetectorElement& self) {
```

Comment:
Does this need to return the shared pointer? Otherwise I'd stick to the C++ return type. 

Is the base class exposed to Python as well? I wonder if it would be enough to expose this method on the base class.

Reply:
I think a shared ptr is required to avoid livetime issues, right?

And you're right, I forgot to pull in the binding of the base class. I think its not strictly necessary, but I think it could be useful to not loose the type information here...